data/campaigns/World_Conquest/resources/strings/mainline.cfg

```diff
@@ -1,90 +1,40 @@
-#textdomain wesnoth-wc
 
-#define WC_STR_PER_LEVEL
-_ "per level" #enddef
+#textdomain wesnoth-wc
 
 #################################
 #textdomain wesnoth-help
 #################################
 
-#define WC_STR_ALWAYS_REST
-_ "Always rest heals" #enddef
-
 #define WC_STR_AMBUSH
 _ "ambush" #enddef
 
-#define WC_STR_BACKSTAB
-_ "backstab" #enddef
-
 #define WC_STR_CASTLE
 _ "Castle" #enddef
 
 #define WC_STR_CAVE
 _ "Cave" #enddef
 
-#define WC_STR_CHARGE
-_ "charge" #enddef
-
 #define WC_STR_CONCEALMENT
 _ "concealment" #enddef
 
 #define WC_STR_CURES
-_ "heals +8"+", "+_ "cures" #enddef
-
-#define WC_STR_DEFENSE
-_ "Defense" #enddef
+_ "heals +8, cures" #enddef
```

Comment:
Is this actually a core string?

Reply:
Ah, yeah, that should be in wesnoth-wc.

Reply:
…does it even belong in this file then?

Reply:
Well, it was in this file before.

Reply:
Sure, but that was when it was a mainline string. Now it's not.

Reply:
I mean feel free to move it somewhere else if you want.